### PR TITLE
mopidy: update to 3.2.0.

### PR DESCRIPTION
**Pull request by TheEgghead27**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64 (glibc))
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - armv7l (crossbuild)

#### Have the results of the proposed changes been tested?
- I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me

### CODE REVIEW

File: srcpkgs/mopidy/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'mopidy'
 pkgname=mopidy
-version=3.0.1
+version=3.2.0
 revision=4
```
Comment (paper42) [inline srcpkgs/mopidy/template:4]:
```suggestion
revision=1
```

Comment (TheEgghead27) [reply srcpkgs/mopidy/template]: I added this change to the new revision :+1:

File: srcpkgs/mopidy/template [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 build_style=python3-module
 hostmakedepends="python3-setuptools python3-Sphinx python3-sphinx_rtd_theme python3-pykka"
 depends="gst1-python3 gst-plugins-good1 gst-plugins-ugly1 gst-plugins-bad1
- python3-tornado python3-pykka python3-setuptools python3-requests python3-uritools"
+ python3-dbus python3-gobject python3-tornado python3-pykka python3-setuptools python3-requests"
+checkdepends="${depends} python3-pytest python3-responses"
 short_desc="Music server for MPD, Spotify, SoundClound and others"
 maintainer="allan <mail@may.mooo.com>"
 license="Apache-2.0"
 homepage="http://www.mopidy.com"
```
Comment (paper42) [inline srcpkgs/mopidy/template:14]:
```suggestion
homepage="https://www.mopidy.com"
```

File: srcpkgs/mopidy/template [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
-version=3.0.1
-revision=4
+version=3.2.0
+revision=1
 wrksrc="Mopidy-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools python3-Sphinx python3-sphinx_rtd_theme python3-pykka"
 depends="gst1-python3 gst-plugins-good1 gst-plugins-ugly1 gst-plugins-bad1
- python3-tornado python3-pykka python3-setuptools python3-requests python3-uritools"
+ python3-dbus python3-gobject python3-tornado python3-pykka python3-setuptools python3-requests"
```
Comment (paper42) [inline srcpkgs/mopidy/template:9]: wrap at 80 columns

File: srcpkgs/mopidy/template (hunk 2 above)
Comment (TheEgghead27) [reply srcpkgs/mopidy/template]: Added :+1:

File: srcpkgs/mopidy/template (hunk 3 above)
Comment (TheEgghead27) [reply srcpkgs/mopidy/template]: Done :+1:

**Pull request merged**